rest:
- 0.00043807243581857374
- -0.843718318704973
- -0.06029983516220461
- -2.353897080760941
- 0.006887294788177222
- 1.5060301684008703
- 2.4253476275735837
approach:
- 0.0012280409148232134
- 0.1428301504926598
- -0.3565530139981654
- -2.5142622235761602
- 0.004023249509847827
- 2.5682742536065737
- 2.305519750573155
grasp:
- 0.2536245021067168
- 0.5389453754578661
- -0.3287808718848646
- -2.333021981924847
- 0.2229654447303878
- 2.9045575537946484
- 1.4292387084741558
hover:
- 0.06607828092156794
- -0.5543255666431574
- -0.33294643360229736
- -3.038527783009044
- -0.17812739281537096
- 2.4117160786222964
- 1.6894966166045347
cold_release:
- 0.7897492558007652
- -0.19881707523371045
- -0.31534752116286957
- -2.94691218129278
- -0.14023996796516683
- 2.793641177680757
- 1.480342807429946
warm_release:
- 0.6562903310332382
- 0.17553549249548658
- -0.33102267230184457
- -2.585935316652956
- -0.028098609242174357
- 2.844442384905285
- 1.2918157128893029
hot_release:
- 0.5608597173992687
- 0.6700208511412988
- -0.3494993938312196
- -1.8075800672330353
- 0.16285494507021372
- 2.626392718824459
- 1.2916530462705498
